docker/mainnet/sbtc-signer/signer-config.toml.in

```diff
@@ -149,6 +149,22 @@ bitcoin_block_horizon = 3000
 # Environment: SIGNER_SIGNER__PROMETHEUS_EXPORTER_ENDPOINT
 prometheus_exporter_endpoint = "0.0.0.0:9184"
 
+# When defined, the signer will attempt to re-run DKG after the specified
+# Bitcoin block height. Please only use this parameter when instructed to by
+# the sBTC team.
+#
+# Required: false
+# Environment: SIGNER_SIGNER__DKG_RERUN_BITCOIN_HEIGHT
+dkg_rerun_bitcoin_height = 880088
+
+# When defined, the signer will attempt/allow multiple rounds of DKG until the
+# specified number of rounds have been completed. Please only use this parameter
+# when instructed to by the sBTC team.
+#
+# Required: false
+# Environment: SIGNER_SIGNER__DKG_ROUNDS_TARGET
+dkg_rounds_target = 2
```

Comment:
nit: target rounds instead of rounds target?

Reply:
[200ca30](https://github.com/stacks-network/sbtc/pull/1204/commits/200ca30ba7e6f808dbaf987571e3fdec5b46c1e7)